docs/regression_test_api.rst

```diff
@@ -87,6 +87,40 @@ For instance, continuing with the example above, one could override the :func:`_
        This only has an effect if used with ``inherit_params=True``.
 
 
+.. py:function:: reframe.core.pipeline.RegressionTest.var(name, *types, value=None, field=None)
+
+   Inserts a new regression test variable.
+   If the ``value`` argument is not provided, the variable is considered *declared* but not *defined*.
```

Comment:
This might confuse the readers. I would probably omit it. And we should say that the `value` is the default value assigned to it.